hydra/_internal/defaults_list.py

```diff
@@ -69,7 +69,12 @@ def __init__(self, repo: IConfigRepository, overrides_list: List[Override]) -> N
             if is_dict or not is_group:
                 self.config_overrides.append(override)
             else:
-                if override.is_delete():
+                if override.is_force_add():
```

Comment:
Not actually related to this PR, but you could gain some indentation back by using an`elif` here

Reply:
yup, done.